packages/web/components/bridge/immersive/use-bridge-quotes.ts

```diff
@@ -560,14 +533,14 @@ export const useBridgeQuotes = ({
 
     if (!transactionRequest || !quote) return;
 
-    if (transactionRequest.type === "evm") {
-      await handleEvmTx({ ...quote, transactionRequest });
-    } else if (transactionRequest.type === "cosmos") {
-      await handleCosmosTx({
-        ...quote,
-        transactionRequest,
-      });
-    }
+    const tx =
+      transactionRequest.type === "evm"
+        ? handleEvmTx({ ...quote, transactionRequest })
+        : handleCosmosTx({ ...quote, transactionRequest });
+
+    await tx.catch((e) => {
+      console.error(transactionRequest.type, "transaction failed", e);
+    });
```

Comment:
I believe we should also throw the error here so it can be caught at the consumer level